src/components/CookieConsent.astro

```diff
@@ -0,0 +1,157 @@
+<div class="consent-banner">
+  <div class="consent-info">
+    <p>Can we store cookies?</p>
+    <p>
+      By clicking “Accept”, you agree to the storing of cookies on your device
+      to analyse site usage.
+    </p>
+  </div>
+  <div class="consent-controls">
+    <a href="/privacy">Privacy Policy</a>
+    <button class="button">Reject</button>
+    <button class="button">Accept</button>
+  </div>
+</div>
+
+<script>
+  const consent = window.localStorage.getItem("cookieConsent");
+  const consentBanner = document.querySelector(
+    ".consent-banner"
+  ) as HTMLDivElement;
+  const acceptButton = document.querySelector(
+    ".consent-banner button:nth-of-type(2)"
+  ) as HTMLButtonElement;
+  const rejectButton = document.querySelector(
+    ".consent-banner button:nth-of-type(1)"
+  ) as HTMLButtonElement;
+
+  const createAdsScript = () => {
+    const googleTag = document.createElement("script");
+    const dataLayer = document.createElement("script");
+    const partyTownScript = document.createElement("script");
+
+    googleTag.src = "https://www.googletagmanager.com/gtag/js?id=G-ES429Z5W7T";
+    googleTag.async = true;
+    googleTag.type = "text/partytown";
+
+    dataLayer.type = "text/partytown";
+    dataLayer.innerHTML = `
+    window.dataLayer = window.dataLayer || [];
+    function gtag() {
+      dataLayer.push(arguments);
+    }
+    gtag("js", new Date());
+    gtag("config", "G-ES429Z5W7T");
+    `;
+
+    partyTownScript.type = "text/partytown";
+
+    document.head.insertBefore(partyTownScript, document.head.firstChild);
+    document.head.insertBefore(googleTag, document.head.firstChild);
+    document.head.insertBefore(dataLayer, document.head.firstChild);
+
+    window.dispatchEvent(new CustomEvent("ptupdate"));
+  };
+
+  if (consent === "true") {
+    consentBanner.remove();
+    createAdsScript();
+  }
+
+  if (consent === "false") {
+    consentBanner.remove();
+  }
+
+  acceptButton.addEventListener("click", () => {
+    window.localStorage.setItem("cookieConsent", "true");
+    consentBanner.remove();
+    createAdsScript();
+  });
+
+  rejectButton.addEventListener("click", () => {
+    window.localStorage.setItem("cookieConsent", "false");
+    consentBanner.remove();
+  });
+</script>
+
+<style>
+  .consent-banner {
+    position: fixed;
+    max-width: 70%;
+    margin: 0 auto;
+    padding: 1rem 2rem;
+    bottom: 10px;
+    left: 0;
+    right: 0;
+    background-color: #fff;
+    display: flex;
+    justify-content: space-between;
+    align-items: center;
+    z-index: 5;
+    border-radius: 5px;
+    border: 1px solid var(--color-primary);
+  }
+
+  .consent-controls {
+    display: flex;
+    align-items: center;
+    gap: 2rem;
+  }
+
+  .consent-info {
+    display: flex;
+    flex-direction: column;
+    gap: 0.5rem;
+    max-width: 40%;
+  }
+
+  .consent-info :first-child {
+    font-size: var(--font-size-base);
+    line-height: 30px;
+  }
```

Comment:
does it work if we let here the default line height and font size for the paragraph? the bold should make it stand out